### PR TITLE
Feat: uses exported `PageProvider` functions and types 

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

Updates versions so that we can use `PageProvider` types and functions in starter.store.
- `@faststore/core` version from https://github.com/vtex/faststore/pull/1994
- `@faststore/cli` version from https://github.com/vtex/faststore/pull/1981

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,18 +7,22 @@
...
     "next": "^12.3.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
     "@cypress/code-coverage": "^3.9.10",
-    "@faststore/cli": "2.1.79",
+    "@faststore/cli": "https://pkg.csb.dev/vtex/faststore/commit/88110a1a/@faststore/cli",
     "@faststore/lighthouse": "^2.1.31",
+    "@graphql-codegen/cli": "2.16.4",
```
Comment (gvc) [inline package.json:23]: Why were these added?

Comment (eduardoformiga) [reply package.json:23]: It's need while running

Screenshot 2023-09-04 at 12 33 04

I'm using the CLI version from 
- https://github.com/vtex/faststore/pull/1981
@icazevedo, maybe we need to add these libs over there?

Comment (icazevedo) [reply package.json:23]: Hmmm I've set them as regular dependencies instead of development dependencies. I'll test this.

**Comment by hellofanny** [comment]
@eduardoformiga I believe we can close this PR since the changes is already merged when updated package to [2.2.0], right?
